### DISCUSSION TITLE
windows, ssl cert

### DISCUSSION THREAD

**Discussion by MOM365** [petrovvlad/freeLib#105] [Ideas]
Привет Влад!
У меня пара вопросов:
1. почему вы не хотите параллельно вести аналог вашего форка freeLib для windows?
Ведь не всегда и не у всех есть возможность выделять сервер для opds/http на Linux. 
2. планируется ли поддержка ssl сертификатов? Сейчас использую старую версию freeLib 5.0.1 под windows,  без поддержки https, что не очнь хорошо.

**Comment by petrovvlad** [comment]
1. Нет необходимости делать отдельный форк для windows. Нужно допилить этот, что бы нормально собирался в windows. Но у меня пока на это нет ни времени, ни желания. 
2. Пока не планирую. Для https рекомедую использовать reverse proxy.
